resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -464,7 +465,7 @@ private[spark] class ApplicationMaster(
       val dummyRunner = new ExecutorRunnable(None, yarnConf, _sparkConf, driverUrl, "<executorId>",
         "<hostname>", executorMemory, executorCores, appId, securityMgr, localResources,
         ResourceProfile.DEFAULT_RESOURCE_PROFILE_ID)
-      dummyRunner.launchContextDebugInfo()
+      log"${MDC(LogKey.LAUNCH_CONTEXT_DEBUG_INFO, dummyRunner.launchContextDebugInfo())}"
```

Comment:
I'm not sure if I want to expand the variables in `dummyRunner.launchContextDebugInfo()`, or is this acceptable?
https://github.com/apache/spark/blob/61dc9d991373c01d449a8ed26d9bfd7eb93f9301/resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ExecutorRunnable.scala#L71-L86

Reply:
This one seems long. Let's expand it.

Reply:
Done.